tests/runtime/test_xgboost.py

```diff
@@ -0,0 +1,44 @@
+import dask.dataframe as dd
+import pytest
+
+# `coiled-runtime=0.0.3,0.0.4` don't contain `xgboost`
+xgb = pytest.importorskip("xgboost")
+
+
+def test_xgboost_distributed_training(small_client):
+    # `coiled-runtime=0.0.3,0.0.4` don't contain `dask_ml`
+    dask_ml = pytest.importorskip("dask_ml")
```

Comment:
@jrbourbeau Is there a reason behind this `importorskip` being inside the tests instead of outside?

Reply:
This isn't strictly needed now, but given this module is for `xgboost`-related test, I can see us adding more tests in the future that use `xgboost` but not `dask-ml`. In that case we'll still want to skip this test, but run other ones that don't rely on `dask-ml`. So putting this `importorskip` inside the test is a bit forward looking

Reply:
Oh I see, that makes sense! Thanks for the explanation :)